src/components/Admin/AdminUsers/AdminUsersHostForm/AdminUsersHostForm.tsx

```diff
@@ -285,7 +285,7 @@ class AdminUsersHostForm extends React.Component<HostProps, HostFormState> {
   render() {
     const { errors } = this.state;
     const { about, btcWalletAddress, email, firstName, lastName, phoneNumber, profilePicUrl, walletAddress } = errors;
-    const { id, listingCount } = this.props.host;
+    const { id, listingCount } = this.props.host || { id: undefined, listingCount: undefined };
```

Comment:
if the host is `undefined` will it just show `undefined` as the id and listingCount in the admin portal?

Reply:
^there's conditional rendering at the bottom